acos_client/v30/responses.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2014,  Doug Wiegley,  A10 Networks.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import re
+
+import acos_client.errors as ae
+
+
+RESPONSE_CODES = {
+    67371011: {
+        '*': {
+            '*': ae.Exists
+        }
+    },
+    1023410176: {
+        'DELETE': {
+            '*': None
+        },
+        '*': {
+            '*': ae.NotFound
+        }
+    },
+    1023410181: {
+        'DELETE': {
+            '*': None
+        },
+        '*': {
+            '/axapi/v3/slb/service-group/.*/member/': ae.NotFound,
+            '*': ae.NotFound
+        }
+    },
+    1023475722: {
+        '*': {
+            '*': ae.NotFound
+        }
+    },
+    1207959957: {
+        '*': {
+            '*': ae.NotFound
+        }
+    },
+}
+
+
+def raise_axapi_auth_error(response, method, api_url, headers):
+    if 'authorizationschema' in response:
+        code = response['authorizationschema']['code']
+        s = response['authorizationschema']['error']
+        if code == 401:
+            if headers and 'Authorization' in headers:
```

Comment:
You could be more explicit with `if headers is not None and 'Authorization' in headers:`. I believe that None is always what your expecting `headers` to be in this case unless it's populated? If not, please disregard.